src/Components/DailyWeather/DailyWeatherCard.jsx

```diff
@@ -0,0 +1,45 @@
+import { getWeatherSymbol } from '../../Helpers/functions';
+import { WiStrongWind } from 'react-icons/wi';
+
+import styles from './DailyWeatherCard.module.scss';
+
+export default function DailyWeatherCard({ dailyWeather }) {
+  const formatDate = (dateStr) => {
+    let selectedDate = new Date(dateStr);
+
+    let date = selectedDate
+      .toLocaleString('en-GB', {
+        day: 'numeric',
+        weekday: 'short',
+        month: 'numeric'
+      })
```

Comment:
you can create shared utility function to get formatted date and use it both in CurrWeatherCard and WeatherCard components

Reply:
done